src/metabase/pulse/render/body.clj

```diff
@@ -872,9 +872,20 @@
       [:img {:style (style/style {:display :block :width :100%})
              :src   (:image-src image-bundle)}]]}))
 
+(defn- get-col-by-name
```

Comment:
`some` might read a little cleaner, and you don't need to traverse the entire collection:

```clojure
(let [cols [{:name "A"}
            {:name "B"}
            {:name "C"}]
      col-name "B"]
  (->> (map-indexed (fn [idx m] [idx m]) cols)
       (some (fn [[idx col]]
               (when (= col-name (:name col))
                 [idx col])))))
```